otj-server-core/src/main/java/com/opentable/server/OTJettyServletWebServerFactory.java

```diff
@@ -52,11 +57,22 @@ public OTJettyServletWebServerFactory() {
     }
 
     @Override
-    protected org.eclipse.jetty.webapp.Configuration[] getWebAppContextConfigurations(WebAppContext webAppContext,
-                                                                                      ServletContextInitializer... initializers) {
-        webAppContextCustomizers.ifPresent(consumers -> consumers.forEach(c -> c.accept(webAppContext)));
-        webAppContext.setErrorHandler(new ConservedHeadersJettyErrorHandler(webAppContext.getErrorHandler(), showStacks));
-        return super.getWebAppContextConfigurations(webAppContext, initializers);
+    protected Configuration[] getWebAppContextConfigurations(WebAppContext webAppContext,
+                                                             ServletContextInitializer... initializers) {
+        List<Configuration> res = new ArrayList<>(Arrays.asList(super.getWebAppContextConfigurations(webAppContext, initializers)));
+        res.add(new AbstractConfiguration() {
+            @Override
+            public void configure(WebAppContext context) {
+                webAppContextCustomizers.ifPresent(consumers -> consumers.forEach(c -> c.accept(context)));
+            }
+        });
+        res.add(new AbstractConfiguration() {
```

Comment:
Note about this configuration has to be the last one given that it is possible in the previous step we could be setting error handler, as is the case for the front door.